src/telemetry/metrics.ts

```diff
@@ -60,6 +60,10 @@ const counterMap = {
       name: 'cio_events',
       description: 'How many customerio events were sent to analytics',
     },
+    cdcTrigger: {
+      name: 'cdc_trigger',
+      description: 'How many times the cdc trigger was called',
+    },
```

Comment:
Wrong place, it's a worker, so it goes into `background`

Reply:
My bad, changed it now.